lenovo/legion/16ach6h/nvidia/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, ... }:
```

Comment:
```suggestion
# This module configures the hardware to run in non-hybrid mode meaning that the amdgpu driver is disabled and only the nvidia card is  used. Thinkpad calls this state "DDG". This module will unlike the default not enable nvidia prime.
{ lib, ... }:
```

Reply:
DDG is a hardware feature to shield amd igpu on hardware and directly connect built-in display to dgpu
This configuration file is used when the “DDG” function is enabled

Reply:
Ok I just wrote a new intro for this module.